### PR TITLE
Upgrade .NET SDK to 2.2.110

### PR DESCRIPTION
**Pull request by AnatoliB**
Upgrade to the latest **LTS** .NET Core 2.2 SDK

### CODE REVIEW

File: tools/helper.psm1 [hunk 1]
```diff
@@ -7,7 +7,8 @@ using namespace System.Runtime.InteropServices
 
 $IsWindowsEnv = [RuntimeInformation]::IsOSPlatform([OSPlatform]::Windows)
 $RepoRoot = (Resolve-Path "$PSScriptRoot/..").Path
-$MinimalSDKVersion = '2.2.0'
+$MinimalSDKVersion = '2.2.110'
```
Comment (Francisco-Gamino) [inline tools/helper.psm1:10]: Question: For .NET `v2.2.8`, based on the guidance here: https://dotnet.microsoft.com/download/dotnet-core/2.2, we could also install `2.2.207`, is that correct?

Comment (AnatoliB) [reply tools/helper.psm1:10]: `2.2.207` is the latest for **VS2019**, but `2.2.110` is the latest for **VS2017**. In the v2.x branch, we are still using **VS2017**. (We could upgrade to **VS2019** like we did with v3.x, but I'm a bit reluctant to do this for a legacy branch, unless there is a strong reason.)

Comment (Francisco-Gamino) [reply tools/helper.psm1:10]: Ok, let's keep it `2.2.110` and continue to use **VS2017**.

**Review by Francisco-Gamino — APPROVED** [review]
LGTM

**Pull request merged**